portfolio/src/main/webapp/General.css

```diff
@@ -0,0 +1,64 @@
+/**
+ Creates a back-button  
+*/
+a:hover {
+    background-color:white;
+    color: black;
+}
+
+.padding {
+    display: inline-block;
+    padding: 8px 16px;
+}
+
+.previous {
+    background-color: rgb(177, 177, 177);
+    color: black;
+}
+
+.round {
+    border-radius: 50%;
+}
+
+/**
```

Comment:
Same for this comment (also capitalize the start)

Reply:
fixed